src/braket/circuits/gates.py

```diff
@@ -109,6 +109,12 @@ def h(
         Returns:
             Iterable[Instruction]: `Iterable` of H instructions.
 
+        Unitary matrix:
```

Comment:
I think this might work if you put it below the examples section.

Reply:
Thanks @krneta. I decided to move it to the top instead, so that it shows the matrix immediately.